onnx_xla/backend.cc

```diff
@@ -187,37 +203,69 @@ namespace onnx_xla {
     value_to_op_[v] = std::move(builder_.GetTupleElement(op, index));
   }
 
-  void XlaTransform::fillIOMetadata()  {
-    executor_->num_inputs_ = (uint32_t) ((int64_t) (ir_->inputs().size()) - (int64_t) (ir_->initializers().size()));
-    executor_->num_outputs_ = (uint32_t) ir_->outputs().size();
+  void XlaTransform::handleInputs()  {
+    ONNX_ASSERT(ir_->initializers().size() == 0 || !weight_descriptors_);
```

Comment:
If this condition fails, ONNXIFI implementation must return an error, not crash the whole process via `abort`

Reply:
support for returning status added